time-macros/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "time-macros"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
This is [your commit](115e447b2641887f03a4b8359923c7427cc93aea). :slightly_smiling_face: Looks like you may have rebased the `v0.3` branch since I rebased onto it, or something like that. Would you like me to rebase this to somewhere new?